tensorzero-internal/src/variant/dicl.rs

```diff
@@ -50,6 +54,26 @@ pub struct DiclConfig {
     pub retries: RetryConfig,
 }
 
+impl Default for DiclConfig {
+    fn default() -> Self {
+        Self {
+            weight: Default::default(),
+            embedding_model: Default::default(),
+            k: Default::default(),
+            model: Default::default(),
+            system_instructions: Default::default(),
+            temperature: Default::default(),
+            top_p: Default::default(),
+            presence_penalty: Default::default(),
+            frequency_penalty: Default::default(),
+            max_tokens: Default::default(),
+            seed: Default::default(),
+            json_mode: JsonMode::Strict,
```

Comment:
this should also be optional and have the same behavior as chat completions (default to None so it can get handled appropriately for the function being used)